packages/proxy-container/tsup.config.ts

```diff
@@ -0,0 +1,3 @@
+import { createTsupConfig } from '../../tsup.config';
+
+export default createTsupConfig({ minify: true, dts: false });
```

Comment:
We should probably write a TODO specifying that we need to setup the options to bundle node_modules, so the Docker image is as small as possible.